packages/app-admin/src/styles/theme.scss

```diff
@@ -27,50 +27,50 @@ $webiny-theme-dark-text-hint-on-dark: rgba(255, 255, 255, 0.15);
 $webiny-theme-dark-caret-down: url("data:image/svg+xml,%3Csvg%20width%3D%2210px%22%20height%3D%225px%22%20viewBox%3D%227%2010%2010%205%22%20version%3D%221.1%22%20xmlns%3D%22http%3A%2F%2Fwww.w3.org%2F2000%2Fsvg%22%20xmlns%3Axlink%3D%22http%3A%2F%2Fwww.w3.org%2F1999%2Fxlink%22%3E%0A%20%20%20%20%3Cpolygon%20id%3D%22Shape%22%20stroke%3D%22none%22%20fill%3D%22%23FFFFFF%22%20fill-rule%3D%22evenodd%22%20opacity%3D%220.54%22%20points%3D%227%2010%2012%2015%2017%2010%22%3E%3C%2Fpolygon%3E%0A%3C%2Fsvg%3E");
 
 // webiny typography
-$webiny-theme-typography-font-family: "Source Sans Pro" !default;
+$webiny-theme-typography-font-family: inherit !default;
 $webiny-theme-typography-subtitle2-font-weight: 600 !default;
 
 // import required fonts
-@import url("https://fonts.googleapis.com/css?family=Source+Sans+Pro:400,600,700|Open+Sans:300,400,600,700|Source+Code+Pro:400,700");
+//@import url("https://fonts.googleapis.com/css?family=Source+Sans+Pro:400,600,700|Open+Sans:300,400,600,700|Source+Code+Pro:400,700");
```

Comment:
Do we want to clean these lines fully maybe?

Or leave them commented out? If so, I'd add a reminder in GH project.

Reply:
Thank you for the suggestion: I have added a reminder in our GH project. 

We need to remove all Material Design tokens from our project files. This can be done while "soft refactoring" the admin app, after we finish developing the components library.